### ISSUE TITLE
quick table scrolling freeze

### ISSUE THREAD

**Issue by Berry669** [ag-grid/ag-grid#5264]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```

**Current behavior**
When scrolling quickly, the table freezes and further scrolling is possible only after a small time delay or after moving the mouse. This behavior is not observable if I make one mouse wheel movement, but if I make several in quick succession. It only occurs if the mouse is positioned in the table, if the mouse is positioned over the scrollbar while scrolling then everything is done without freezing. I have a feeling that the hover effect on the table row has something to do with it. The behavior can be observed on the official examples on aggrid's website, e.g. here: https://www.ag-grid.com/

**Please tell us about your environment:**
windows 11

* **AG Grid version:** X.X.X
ag-grid-community: 27.3.0
ag-grid-vue3: 27.3.0

* **Browser:**
chrome, edge

**Comment by makinggoodsoftware** [comment]
Hi,

Note that this seems to be the same issue as this one:

https://github.com/ag-grid/ag-grid/issues/5243

Going to close this one

**Comment by gportela85** [comment]
This is an issue with Chrome, see: https://bugs.chromium.org/p/chromium/issues/detail?id=1330045
They have patched this bug in Chrome Canary v104.0.5108, but it might be a while until v104 is officially released.

Reported here by: https://github.com/ag-grid/ag-grid/issues/5243
